src/AzureAuth/CommandMain.cs

```diff
@@ -434,13 +434,15 @@ private bool ValidateOptions()
                 validOptions = false;
             }
 
+#if PlatformWindows

```

Comment:
I'm wary of using conditional compilation to get out of this issue. I think there might be a logic bug we can solve elsewhere. If we can, let's explore that angle before we merge this.

Reply:
Removed `--cache` and all its validation code.